source/class/qx/ui/form/ToggleButton.js

```diff
@@ -114,6 +114,16 @@ qx.Class.define("qx.ui.form.ToggleButton", {
       apply: "_applyTriState",
       nullable: true,
       init: null
+    },
+
+    /**
+     * The behavior when the button is clicked. Only useful for tri-state checkboxes.
+     * If "cycle" is set, the button cycles through the states "disabled", "undetermined", and "enabled"
+     * If "toggle" is set, the button toggles between "disabled" and "enabled".
+     */
+    clickBehavior: {
```

Comment:
This property name is a bit of a misnomer, since it is applied whether the button is clicked or toggled elsewise, e.g., via the keyboard. Can we rename it to `executeBehavior` instead (and fix the doc comment too)?

BTW, this'll be a nice new feature.